src/main.py

```diff
@@ -97,5 +109,17 @@ def main():
     print('Finished Training')
 
 
+def loadWeights(model):
+    cwd = os.getcwd()
+    PATH = cwd + "model.pkl"
+    model = torch.load(PATH)
+
+
+def saveWeights( model):
+    cwd = os.getcwd()
+    PATH = cwd + "model.pkl"
+    torch.save(model, PATH)
+
+
```

Comment:
for line `112 - 121`, I believe what you are doing is to try to save a trained model here ?

Reply:
Yeah, I wanted to ads some saving and resuming mechanism, I did not want the pull request to include that commit 🙁